### ISSUE TITLE
Unclear comments in telescope_model.py

### ISSUE THREAD

**Issue by GernotMaier** [gammasim/gammasim-tools#184]
https://github.com/gammasim/gammasim-tools/blob/20a1e9362cf3a5d5621b57bf647d9e18c09b0667/simtools/model/telescope_model.py#L284

It says 
```
        # UPDATE: SimtelConfigWriter delas with which parameters should be written or not.
        # _parameters here can contain all the parameters from the DB.
```

- why is it about writing, when the function is called  _loadParametersFromDB ?
- Why does 'contain all parameters from the DB' mean. Really all parameters, or only those for the requested site and model version? I assume the later?

**Comment by RaulRPrado** [comment]
The thing is that not all the parameters that we store on the DB should be written to the sim_telarray config file.

You can see https://github.com/gammasim/gammasim-tools/blob/3c061f791f34701f5906b8afb385d67eb8c73c06/simtools/simtel/simtel_config_writer.py#L39-L49 for the list of parameters that we read from the DB but not write to the config file.

This was added later to decoupled the telescope_model from the sim_telarray specifics. I am not sure this comment is actually necessary anymore.

**Comment by GernotMaier** [comment]
Ok, that makes sense. But none of that is reflected by your comment and cannot by understood correctly.

May I cite from a book you recommended me? 'The proper use of comments is to compensate for our failure to express ourselves in code'. This is what I mean: the function is called _load... and your comments discuss writing. (and you shouldn't have recommend me that book; I look differently at code; especially at my own)

**Comment by RaulRPrado** [comment]
Glad to see the book is being relevant =)

You are right. I think I was talking to myself in this comment.
For now, I would just remove it. The fact that the DB contains parameters that are not written in the config file should be a well known (and documented) fact for developers.

**Comment by RaulRPrado** [comment]
Fixed in b1f7575ce572cc92ad415889c613cad11c435822
